Dockerfile

```
@@ -1,5 +1,9 @@
 FROM rockylinux:9
-MAINTAINER APEL Administrators <apel-admins@stfc.ac.uk>
+LABEL org.opencontainers.image.authors="apel-admins@stfc.ac.uk" \
+        org.opencontainers.image.title="APEL SSM" \
+        org.opencontainers.image.description="Secure Stomp Messenger (SSM) is designed to simply send messages using the STOMP protocol." \
```

```suggestion
        org.opencontainers.image.description="Secure STOMP Messenger (SSM) is designed to simply send messages using the STOMP protocol or via the ARGO Messaging Service (AMS)." \
```

That's the wording in the README.md, and I would say it makes sense to be consistent.